ts/src/okx.ts

```diff
@@ -2825,6 +2826,15 @@ export default class okx extends Exchange {
                 request['slTriggerPxType'] = slTriggerPxType;
             }
         }
+        // for users convenience, automatically set `posSide` (in "hedge" mode) if not set
+        const isSLTP = (takeProfitPrice !== undefined) || (stopLossPrice !== undefined) || (trigger && isReduceOnly);
```

Comment:
two things missing, a test and adding `hedged` to JSDocs

Reply:
yes, as said in top-post, at first i wanted your confirmation on this. if so, i'll add them now.

Reply:
sorry I missed that, a few more questions,

what if it's not a tp sl orders, does not the same apply?

if so, what if the order is a reduce only?

Reply:
actually, i was only thinking for sl/tp orders because of the user who originally reported the issue, but now I think that you are right, we can handle those cases too, to simplify for users:
a) for regular (non SL/TP) orders it's also needed, yes
b) for regular (non Trigger) reduceOnly it's also needed, yes

will re-push soon.

Reply:
reduceOnly orders not yet supported, while TP/SL supported (crazy, but that's case) so I've temproarily commented reduceonly case.

Reply:
@carlosmiei I think it's finished for now.